add tidbit about when findings are created (#1962)
semgrep/semgrep-docs: docs/semgrep-appsec-platform/jira.md

- You can automate the creation of tickets for high severity, high confidence findings in your [default branch](/deployment/add-semgrep-to-ci#default-branch-names).
  - Automated ticket creation can be configured on a per-product setting.
  - For Supply Chain findings, tickets are created for reachable findings.

>   - Tickets are automatically created for **net new** findings from subsequent scans after enabling automated ticket creation. Tickets are not automatically created for existing findings.

Reviewer: ```suggestion
  - Tickets are automatically created for **new** findings identified by scans that occur after enabling automated ticket creation. Tickets are not automatically created for existing findings.
```
